### ISSUE TITLE
Last.fm scrobbling stopped working

### ISSUE THREAD

**Issue by sigsergv** [tiliado/nuvolaplayer#172]
I'm using nuvolapayer on debian jessie and ubuntu 14.04. And last.fm scrobbling has stopped after upgrade, player just doesn't send anything. Debug mode is not helping much, it doesn't mention last.fm at all.

I've used wireshark to monitor traffic (ws.audioscrobbler.com) and captured just a single request — when the application starts, and nothing else.

Packages list:
~~~~~
ii  nuvolaplayer3                                     3.0.0~201510101914.d95d0c6-0+1~jessie 
ii  nuvolaplayer3-8tracks                             5.0-1~jessie                          
ii  nuvolaplayer3-all-services                        0.201509260836-1~jessie               
ii  nuvolaplayer3-amazon-cloud-player                 5.0-1~jessie                          
ii  nuvolaplayer3-bandcamp                            2.0-1~jessie                          
ii  nuvolaplayer3-deezer                              2.3-1~jessie                          
ii  nuvolaplayer3-google-play-music                   5.7-1~jessie                          
ii  nuvolaplayer3-jango                               2.0-1~jessie                          
ii  nuvolaplayer3-kexp                                1.0-1~jessie                          
ii  nuvolaplayer3-logitech-media-server               2.0-1~jessie                          
ii  nuvolaplayer3-mixcloud                            1.0-1~jessie                          
ii  nuvolaplayer3-plex                                1.1-1~jessie                          
ii  nuvolaplayer3-rdio                                4.1-1~jessie                          
ii  nuvolaplayer3-spotify                             2.1-1~jessie                          
ii  nuvolaplayer3-this-is-my-jam                      2.0-1~jessie                          
ii  nuvolaplayer3-tunein                              1.0-1~jessie                          
ii  nuvolaplayer3-yandex-music                        1.1-1~jessie                          
~~~~~

Debug log: https://gist.github.com/sigsergv/7ca7c850be5c0bcf4742

**Comment by fenryxo** [comment]
Hi @sigsergv. Thanks for your report. This issue may have a same cause as in #171. Could you check whether a freshly released Google Play Music script 5.8 fixes also your issue?

**Comment by sigsergv** [comment]
Yeah, Google Play Music script 5.8 fixes it.

Thanks, I think I close this issue then.
